### DISCUSSION TITLE
Sensor deployment response file

### DISCUSSION THREAD

**Discussion by tommorgan365** [Security-Onion-Solutions/securityonion#1553] [Unsupported Versions]
Hello,
Is there a response file for sensor deployment in the new version?

**Comment by tommorgan365** [comment]
Reviewed the lab 1.2 for the forward node install and that one was manual install.  Looking to deploy a few was looking for a response file to drop in place.

**Reply by weslambert** [reply]
We don't have an official supported response file solution at the moment.
